src/quo/core.cljs

```diff
@@ -157,7 +157,7 @@
 (def group-avatar quo.components.avatars.group-avatar.view/view)
 (def icon-avatar quo.components.avatars.icon-avatar/icon-avatar)
 (def user-avatar quo.components.avatars.user-avatar.view/user-avatar)
-(def wallet-user-avatar quo.components.avatars.wallet-user-avatar/wallet-user-avatar)
+(def user-avatar quo.components.avatars.wallet-user-avatar.view/wallet-user-avatar)
```

Comment:
There's a redefined variable here, is this intended?

Reply:
fixed